gen/remote.go

```diff
@@ -43,7 +43,7 @@ func getRemoteCommit(exercise, token string) (Header, error) {
 	url := fmt.Sprintf(commitsURL, exercise)
 	fmt.Printf("[REMOTE] fetching latest commit (source: %s)\n", url)
 
-	req, err := http.NewRequest("GET", url, nil)
+	req, err := http.NewRequest("GET", url, http.NoBody)
```

Comment:
Is there some linter suggesting this change? In real life, nil is very common to be used here and also mentioned in the official documentation.

Reply:
I think it was a linter suggestion